src/styling/base/_icon.scss

```diff
@@ -1,10 +1,10 @@
 @font-face {
   font-family: "ipwa";
-  src:  url("../fonts/ipwa/ipwa.eot?5gk46f");
-  src:  url("../fonts/ipwa/ipwa.eot?5gk46f#iefix") format("embedded-opentype"),
-    url("../fonts/ipwa/ipwa.ttf?5gk46f") format("truetype"),
-    url("../fonts/ipwa/ipwa.woff?5gk46f") format("woff"),
-    url("../fonts/ipwa/ipwa.svg?5gk46f#ipwa") format("svg");
+  src:  url("../assets/fonts/ipwa/ipwa.eot?5gk46f");
```

Comment:
Rank B
I guess the font icon is missing in the prod env

Reply:
I tested on the prod environment, everything works fine.